src/sagemaker_pytorch_serving_container/ts_environment.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module contains functionality that provides access to system
+characteristics, environment variables and configuration settings.
+"""
+
+from __future__ import absolute_import
+
+from sagemaker_pytorch_serving_container import ts_parameters
+
+import os
+import logging
+
+logger = logging.getLogger()
+
+DEFAULT_TS_BATCH_SIZE = 1
+DEFAULT_TS_MAX_BATCH_DELAY = 100
+DEFAULT_TS_MIN_WORKERS = 1
+DEFAULT_TS_MAX_WORKERS = 1
+DEFAULT_TS_RESPONSE_TIMEOUT = 120
```

Comment:
NIT: IIRC the sagemaker default for `response_timeout` is 60s. We should confirm this with the inference hosting team and set it to the same value.
Also, double check if they set a default value for `max_batch_delay`